uring/jvm/src/main/scala/fs2/io/uring/IOExceptionHelper.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 Arman Bilge
```

Comment:
We should probably update the copyright. :-)

Reply:
its minez 😏

Reply:
😂